homeassistant/helpers/script.py

```diff
@@ -274,6 +274,9 @@ async def async_stop(self) -> None:
         await self._stopped.wait()
 
     def _log_exception(self, exception):
+        if self._script.domain == "automation":  # don't use const: circular import
```

Comment:
Thinking about this more, I think that we should remove all logging from the script helper. Instead it should always be handled by the caller of the script helper. They know best how to work with this.

I think that we should wrap these exceptions in a new `ScriptError` that has the right action and message set. But that might be a bigger refactor.

Reply:
So I am not sure if we should merge this PR as a stopgap measure after which we'll most likely forget about it and never fix it properly 😬

Reply:
I agree, it would certainly be cleaner to do a larger refactoring. However, the change works (surprisingly well) and we could probably spend our time better then to do a bigger change just to eliminate one extra log message.

Personally I wouldn't mind not changing this at all. With #45613 being merged, both errors are short and should help you identify the issue quickly. I would feel different if Home Assistant would log complete tracebacks every time, especially since these errors are re-raised multiple times.

Reply:
Then let's close it and let's wait for someone to tackle a big refactor.